src/main/java/step1/Operator.java

```diff
@@ -0,0 +1,42 @@
+package step1;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BinaryOperator;
+
+public enum Operator {
+    ADD((num1, num2) -> num1 + num2),
+    SUBTRACT((num1, num2) -> num1 - num2),
+    MULTIPLY((num1, num2) -> num1 * num2),
+    DIVIDE((num1, num2) -> num1 / num2);
+
+    private final static List<String> operators = Arrays.asList("+", "-", "*", "/");
+    private BinaryOperator<Integer> expression;
+    private static Map<String, Operator> operator;
```

Comment:
상수는 `static final` 필드입니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
    private static final Map<String, Operator> OPERATOR;
```

Reply:
감사합니다~!! 사칙 연산에 대한 함수 전달로 더 이상 할당되거나 값이 추가하는 일이 없기에 상수로 보는게 맞는 것 같습니다!!!